doc/man5/srec_gowin.5

```diff
@@ -0,0 +1,56 @@
+'\" t
+.\"     srecord - manipulate eprom load files
+.\"     Copyright (C) 2023 Daniel Anselmi
+.\"
+.\"     This program is free software; you can redistribute it and/or modify
+.\"     it under the terms of the GNU General Public License as published by
+.\"     the Free Software Foundation; either version 3 of the License, or
+.\"     (at your option) any later version.
+.\"
+.\"     This program is distributed in the hope that it will be useful,
+.\"     but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\"     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+.\"     GNU General Public License for more details.
+.\"
+.\"     You should have received a copy of the GNU General Public License
+.\"     along with this program. If not, see
+.\"     <http://www.gnu.org/licenses/>.
+.\"
+.ds n) srec_gowin
+.TH \*(n) 5 SRecord "Reference Manual"
+.SH NAME
+srec_gowin \- Gowin bitstream file format
+.if require_index \{
+.XX "srec_gowin(5)" "Gowin bitstream file format"
+.\}
```

Comment:
- [ ] These three lines from require_index block have to be moved up between .TH and .SH when #58 has been merged.

Reply:
#58 is now merged so now ready for this change.